torch/jit/_recursive.py

```diff
@@ -239,14 +252,6 @@ def infer_type(name, item):
                     "to a TorchScript type.)").format(torch.typename(type(value)))
             concrete_type_builder.add_failed_attribute(name, hint)
 
-    # Add @property methods as failed attributes, to give a better error message.
-    for name, value in type(nn_module).__dict__.items():
-        if isinstance(value, property):
-            hint = ("\n(This attribute exists on the Python module, but it's an @property "
-                    "method. @property methods are not yet supported in TorchScript. "
-                    "Please file a feature request on Github)")
-            concrete_type_builder.add_failed_attribute(name, hint)
```

Comment:
I wasn't quite sure what to do here. Properties are really just syntactic sugar for setter and getter methods. Do they belong on the concrete type? If yes, how do we represent them? If not, what happens if two modules have the same underlying data attributes but different property getters/setters for them?

Reply:
So, concrete type already returns false if two classes differ: pyClass_.is(other.pyClass_). As long as we only take properties from methods, and not from attributes, we don't need to do any additional work here. If we do take properties as attributes than we'll need to add stuff to the ConcreteType. 

(i think it's fine to not take properties that are passed in as attributes, no one really does that)